sdk/src/evaluations/index.ts

```diff
@@ -0,0 +1,82 @@
+import type { TypedDataSigner } from "@ethersproject/abstract-signer";
+import { ethers } from "ethers";
+import { CIDString } from "nft.storage";
+
+import { DEFAULT_CHAIN_ID } from "../constants.js";
+import HypercertsStorage from "../storage.js";
+import { StorageError } from "../types/errors.js";
+import { EASEvaluation, EvaluationSource, HypercertEvaluationSchema, IPFSEvaluation } from "../types/evaluation.js";
+import EasEvaluator from "./eas.js";
+
+const EASContractAddress = "0xC2679fBD37d54388Ce493F1DB75320D236e1815e"; // Sepolia v0.26
```

Comment:
This should probably be part of the SDK config right? It'll be different values depending on the network

Reply:
Yes, but since we're signing off chain it's just to have a pointer to where the schema is stored, we're not actually using it. When we move to on-chain attestations we need to make this dynamic